### PR TITLE
Add fit-content function

### PR DESCRIPTION
**Pull request by rachelandrew**
Working on https://github.com/mdn/content/issues/6715

Adding the information about `fit-content()` to relevant properties.

### CODE REVIEW

File: css/properties/block-size.json [hunk 1]
```diff
@@ -100,6 +100,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (ddbeck) [inline css/properties/block-size.json:157]: First implementations should always be experimental ([guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status)). I think this applies to every feature touched by this PR.

```suggestion
              "experimental": true,
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks, @rachelandrew. One suggestion, affecting all of these features.

File: css/properties/block-size.json (hunk 1 above)
Comment (ddbeck) [reply css/properties/block-size.json]: Only this one entry was fixed. Did you forget to push the rest?

File: css/properties/height.json [hunk 2]
```diff
@@ -109,6 +109,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/height.json:166]:
```suggestion
              "experimental": true,
```

File: css/properties/max-inline-size.json [hunk 3]
```diff
@@ -112,6 +112,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/max-inline-size.json:169]:
```suggestion
              "experimental": true,
```

File: css/properties/max-width.json [hunk 4]
```diff
@@ -144,6 +144,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/max-width.json:201]:
```suggestion
              "experimental": true,
```

File: css/properties/min-block-size.json [hunk 5]
```diff
@@ -98,6 +98,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/min-block-size.json:155]:
```suggestion
              "experimental": true,
```

File: css/properties/min-height.json [hunk 6]
```diff
@@ -195,6 +195,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/min-height.json:252]:
```suggestion
              "experimental": true,
```

File: css/properties/min-inline-size.json [hunk 7]
```diff
@@ -100,6 +100,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/min-inline-size.json:157]:
```suggestion
              "experimental": true,
```

File: css/properties/min-width.json [hunk 8]
```diff
@@ -210,6 +210,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/min-width.json:267]:
```suggestion
              "experimental": true,
```

File: css/properties/max-height.json [hunk 9]
```diff
@@ -155,6 +155,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/max-height.json:212]:
```suggestion
              "experimental": true,
```

File: css/properties/max-block-size.json [hunk 10]
```diff
@@ -98,6 +98,66 @@
...
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
```
Comment (rachelandrew) [inline css/properties/max-block-size.json:155]:
```suggestion
              "experimental": true,
```

File: css/properties/block-size.json (hunk 1 above)
Comment (rachelandrew) [reply css/properties/block-size.json]: I'd not done anything other than commit your change, I've done the rest now.

**Review by ddbeck — APPROVED** [review]

**Pull request merged**